tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

```diff
@@ -519,42 +551,57 @@ public void expectCompletion(long timeoutMillis, String errorMsg) throws Interru
     }
 
     public <E extends Throwable> void expectErrorWithMessage(Class<E> expected, String requiredMessagePart) throws Exception {
-      expectErrorWithMessage(expected, requiredMessagePart, env.defaultTimeoutMillis());
+      expectErrorWithMessage(expected, Collections.singletonList(requiredMessagePart), env.defaultTimeoutMillis(), env.defaultPollTimeoutMillis);
```

Comment:
If we end up introducing this new value, then it must not be accessed directly like this; 
it would have to gain an 

```
  public long defaultNoSignalsTimeoutMillis() {
    return defaultNoSignalsTimeoutMillis;
  }
```

Similar to  `defaultNoSignalsTimeoutMillis()` because that's a customization point people can use to override and set a value they may want to use there rather via the constructor;

Reply:
I can add a new accessor for `defaultPollTimeoutMillis` if you prefer, note that the accessor was not added to avoid increasing the footprint of this class.